### PR TITLE
PT-580 Rebase from tidepool/platform v1.32.1

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,7 +32,103 @@ The Tidepool platform API
...
+* Update Dexcom types to latest best practices
+* Add data store benchmark tool
+* Rename local variable name context to ctx for consistency
+* Remove unused code in data store mongo implementation
+* Update tool dependencies
+* Add build tools as additional dependencies
+* Update Makefile to install build tools from vendor directory
+* Update Makefile to remove unused tools
+
+## v1.29.0
```
Comment (CedricPerotto) [inline CHANGELOG.md:131]: Do we need the full Tidepool ChangeLog ?

**Review by CedricPerotto — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (yvon-dblg) [reply CHANGELOG.md:131]: I don't know... Maybe something to discuss in a daily ?
Having it may be easier in the future merge.

**Review by CedricPerotto — APPROVED** [review]
LGTM, just bump release version & release date in the Changelog.

**Review by CedricPerotto — APPROVED** [review]
LGTM

**Comment by yvon-dblg** [comment]
Clean PR: https://github.com/mdblp/platform/pull/34

**Pull request closed without merging**